### PR TITLE
Implemented Add Movie to Watchlist

### PR DESCRIPTION
**Pull request by timzprof**
- Implemented Add Movie to Watchlist Endpoint by adding Movie ID to user watchlist array

auth user can add movies to watchlist

### CODE REVIEW

File: src/util/user.js [hunk 1]
```diff
@@ -0,0 +1,10 @@
+// Helper Methods
+export const trimUser = (user) => {
+    return {
+        id: user.dataValues.id,
+        username: user.dataValues.username,
+        email: user.dataValues.email,
+        watchlist: user.dataValues.watchlist,
+        ratedMovies: user.dataValues.ratedMovies
+    };
+}
```
Comment (daniellamarr) [inline src/util/user.js]: new line

File: src/controllers/user.js [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+            const user = await userModel.findOne({where: {email: req.user.email}});
+
+            if(!user){
+                const userNotFoundError = new Error(`User with email - ${req.user.email} Not Found`);
+                userNotFoundError.statusCode = 404;
+                throw userNotFoundError;
+            }
+
+            const updatedUser = await user.update({
+                watchlist: [...user.dataValues.watchlist, req.body.movieId]
```
Comment (daniellamarr) [inline src/controllers/user.js]: Are you verifying for the correct movie ID?

File: src/util/user.js (hunk 1 above)
Comment (timzprof) [reply src/util/user.js]: Added now

File: src/controllers/user.js (hunk 2 above)
Comment (timzprof) [reply src/controllers/user.js]: Now that you mention it, I am not right now
I didn't think there was a way to verify it but an idea just hit me on  how I can do that 

I'll add that now

Comment (daniellamarr) [reply src/controllers/user.js]: All right

File: src/index.js [hunk 3]
```diff
@@ -98,6 +98,7 @@ app.use((error, req, res, next) => {
 		message: "Something went wrong",
 		errorMessage: error.message
 	};
+	console.log(responseObj);
```
Comment (daniellamarr) [inline src/index.js]: Any reason for this console.log?

Comment (timzprof) [reply src/index.js]: Mad o, Sorry I forgot it there 😅😅

**Pull request merged**